using_images/s2i_images/perl.adoc

```diff
@@ -103,8 +103,8 @@ recommended to turn this on in your production environment.
 Hot deployment allows you to quickly make and deploy changes to your application
 without having to generate a new S2I build. To enable hot deployment in this
 image, you must set the `*PERL_APACHE2_RELOAD*` environment variable to *true*.
-You can use
-link:../../dev_guide/environment_variables.html#set-environment-variables[the
+For example, see link:../../dev_guide/new_app.html#specifying-environment-variables[the `oc new-app` command].
+You can use link:../../dev_guide/environment_variables.html#set-environment-variables[the
 `oc env` command] to update environment variables of existing objects.
```

Comment:
Hmm. I think a better solution would be to either ignore the fact you need to hover. Or not using backtics?